Wbooks/src/app/screens/navigation/stackNavigator.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+
+import Library from '../library/components/bookList';
+import BookDetail from '../bookDetail';
+import { defaultNavOptions } from '../../../config/navigationOptions';
+
+import routes from './routes';
+
+const Stack = createStackNavigator();
+
+function StackNavigator() {
+  return (
+    <Stack.Navigator>
+      <Stack.Screen
+        name={routes.Library}
+        component={Library}
+        options={defaultNavOptions('Notifications', 'Search', 'LIBRARY')}
+      />
+      <Stack.Screen
+        name={routes.BookDetail}
+        component={BookDetail}
+        options={defaultNavOptions('Back', null, 'BookDetail')}
+      />
+    </Stack.Navigator>
+  );
+}
```

Comment:
Nice! Have in mind that `StackNavigator` is not a good name for this component because is suppose to be part of your app flow and you will have 2, 3 or more stacks.